scripts/scripts.js

```diff
@@ -1359,8 +1374,10 @@ sampleRUM.drain('convert', (cevent, cvalueThunk, element, listenTo = []) => {
           sampleRUM('variant', { source: experiment, target: treatment });
         });
       // send conversion event
-      const cvalue = typeof cvalueThunk === 'function' ? cvalueThunk() : cvalueThunk;
-      sampleRUM('convert', { source: cevent, target: cvalue, element: celement });      
+      const cvalue = typeof cvalueThunk === 'function' ? cvalueThunk(element) : cvalueThunk;
```

Comment:
Just wondering: we might want to make the entire inner function `async` so that we can support `async` thunks:
```suggestion
      const cvalue = typeof cvalueThunk === 'function' ? await cvalueThunk(element) : cvalueThunk;
```

Reply:
Done